.github/workflows/release.yml

```diff
@@ -38,6 +38,8 @@ jobs:
     steps:
       - name: Checkout cli
         uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.REPO_ACCESS_TOKEN }}
```

Comment:
Is it a custom token variable? Where will this value come from?

Reply:
yes, it is from [secrets](https://github.com/twilio/twilio-cli/settings/secrets/actions).